x-pack/plugins/search_connectors/server/routes/connectors_routes.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CONNECTOR_DEFINITIONS } from '@kbn/search-connectors';
+import { RouteDependencies } from '../plugin';
+
+export const registerConnectorsRoutes = ({ http, router }: RouteDependencies) => {
```

Comment:
Do we plan to migrate other connector routes to this plugin as well?

Reply:
I think in the long term we should migrate routes here and export functions to call them, the way the ML plugin does. Obviously not urgently needed work, but will be useful down the line.